### PR TITLE
add info about postback to message event (request)

### PR DESCRIPTION
**Pull request by bkubiak**

### CODE REVIEW

File: content/messaging/agent-chat-api/index.mdx [hunk 1]
```diff
@@ -351,6 +351,12 @@ The **Message** event contains text message to other chat users.
 | `type`                | yes     | `string`  | `message`                                                           |
 | `recipients`          | yes     | `string`  | Possible values: `all` (default), `agents`                          |
 | `properties`          | no      | `object`  | [Properties](#properties)                                           |
+| `postback`            | no      | `object`  | Indicates that the message event was generated from a rich message event. |
```
Comment (gluszczyk) [inline content/messaging/agent-chat-api/index.mdx]: how about
`Indicates that the message event was generated as the response to a rich message event.`
?

Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/index.mdx]: why as the response? what do you mean?

Comment (gluszczyk) [reply content/messaging/agent-chat-api/index.mdx]: If you ask me a closed question in the form of a rich message with some buttons and I click one of them then the postback will tell you which one I clicked (what my response is).

Comment (ale-kacperczyk) [reply content/messaging/agent-chat-api/index.mdx]: I see. What about: 
Indicates that the message event was generated in response to a rich message event.

I believe the meaning is the same but sounds a bit more natural.

Comment (bkubiak) [reply content/messaging/agent-chat-api/index.mdx]: ok

**Comment by ale-kacperczyk** [comment]
@bkubiak remember to name branches `feature/branch name`, please. It generates a preview on labs :)

**Comment by bkubiak** [comment]
@ale-kacperczyk Ok, I forgot. You have described it here: https://livechatinc.atlassian.net/wiki/spaces/PAT/pages/761529400/Updating+API+docs. Thanks!

**Pull request merged**